### PR TITLE
add tensorflow support link

### PR DESCRIPTION
**Pull request by LisaDelaney**

### CODE REVIEW

File: docs/install/tensorflow-install.md [hunk 1]
```diff
@@ -22,11 +20,14 @@ version support matrix:
 | Post-5.7.0 | Last three versions at ROCm release. |
 :::
 
-### Installing TensorFlow
+For current TensorFlow ROCm version support, refer to the
+[GitHub repository](https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/blob/develop-upstream/rocm_docs/tensorflow-rocm-release.md).
```
Comment (LisaDelaney) [inline docs/install/tensorflow-install.md:24]: @jayfurmanek here is the link update. Do you have a better name for "GitHub repository" for this link?

**Review by Rmalavally — DISMISSED** [review]
Looks good. Thanks so much.

File: docs/install/tensorflow-install.md (hunk 1 above)
Comment (johnnynunez) [reply docs/install/tensorflow-install.md:24]: @saadrahim last version of tensorflow v2.13, not is compiled with 7900xtx

**Review by saadrahim — DISMISSED** [review]

File: docs/install/tensorflow-install.md (hunk 1 above)
Comment (saadrahim) [inline docs/install/tensorflow-install.md:24]:
```suggestion
[ROCm fork of the TensorFlow repository](https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/blob/develop-upstream/rocm_docs/tensorflow-rocm-release.md).
```

**Pull request merged**